index.html

```diff
@@ -46,27 +53,20 @@ <h1 id="title">Survey Form</h1>
         <label for="check" class="input-t">Kotlin</label>
       </div>
       <div class="textarea">
-        <textarea name="review" id="textarea" cols="30" rows="10" placeholder="Comments"></textarea>
+        <textarea
+          name="review"
+          id="textarea"
+          cols="30"
+          rows="10"
+          placeholder="Comments"
+        ></textarea>
       </div>
       <div class="btn">
         <button id="submit">Submit</button>
       </div>
     </form>
 
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
```

Comment:
- [ ] [REQUIRED] Please when giving code review, it is just appropriate to phrase your comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think your codebase would be cleaner if you remove unused code block commented out. Kindly remove the above commented code blocks"*
* This reads as an order: *"Remove comments inside the code blocks to avoid chunks"* 

- [ ] Also, your comment will be more useful if it includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

Reply:
- [ ] Indeed it is crucial to use an accurate and respectful language to communicate with the author of the PR. For this reason, I would suggest using explanatory and friendly phrases like:

    - "I think it would be better to use semantic tags" :heavy_check_mark: 
    - "Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible" :heavy_check_mark: